src/metabase/models/query_execution.clj

```diff
@@ -15,11 +15,10 @@
 (def Context
   "Schema for valid values of QueryExecution `:context`."
   (s/enum :ad-hoc
-          :csv-download
+          :download
           :dashboard
           :embedded-dashboard
           :embedded-question
-          :json-download
```

Comment:
@salsakran do we want to stop differentiating between whether a query is downloaded as CSV/JSON/Excel? Seems like it might be useful for usage stats